newsletter/backend/newsletters/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+
+from . import models
+
```

Comment:
There is a better way to register a model 😉

Reply:
Like here.

https://github.com/AzenoIT/azeno/blob/master/flashcards/backend/decks/admin.py

Reply:
but the way to import the model should be better.

Reply:
import is ok, don't see a problem here, but registering the model in the admin panel is a logical part of creating it so I suggest merging those two tasks ;)

Reply:
Registered model in admin.py as Martyna suggested.